R/cube-result.R

```diff
@@ -151,6 +143,7 @@ takeSelectedDimensions <- function (x, dims) {
     if (any(selecteds)) {
         drops <- lapply(selecteds, function (s) {
             ## For "Selected" dimensions, we only want to return "Selected", the 1st element
+            ## TODO: this could be brittle; consider an is.selected attr/vector
```

Comment:
Not a big deal, but it might be worth adding the ways this might be brittle to the #TODO.